doubly_linked_list/doubly_linked_list.py

```diff
@@ -40,6 +40,24 @@ def __init__(self, node=None):
     self.tail = node
 
   def add_to_head(self, value):
+    if self.head is None and self.tail is None:
+      self.head = ListNode(value)
+      self.tail = self.head
+      self.tail.next = None
```

Comment:
By default, the prev and next attributes of a newly created ListNode object are already set to None, so you probably don't need these lines